c/pthread/bigshot_p_false-unreach-call.c

```diff
@@ -10,12 +10,14 @@ char *v;
 
 void *thread1(void * arg)
 {
-  v = malloc(sizeof(char) * 8);
+  v = calloc(8, sizeof(char));
```

Comment:
Why do we need `calloc` here? We never read the allocated memory, but potentially override it with new values.

Reply:
Then reason is the read in assert at line 33 (https://github.com/paradise-fi/sv-benchmarks/blob/d9d6dd6338b4ed1a817f1e33e1eae3294edcec2f/c/pthread/bigshot_p_false-unreach-call.c#L33), a verifier with detection of branching on initialized memory then reports error other then the reachability of `__VERIFIER_error`.

Reply:
According to the [SV-Comp rules](https://sv-comp.sosy-lab.org/2019/rules.php) the method `malloc` always returns a valid pointer. Thus I would not expect any branching here.
However, the intention of the benchmark is not affected by the change, thus we can keep it.

Reply:
The problem is not validity of the pointer returned by `malloc`, but undefined values in the pointed-to memory. `calloc` does zero-initialize the memory, `malloc` does not.